pkg/serverutils/utils.go

```diff
@@ -25,6 +26,32 @@ func SendResponse(rw http.ResponseWriter, code int, resp interface{}) {
 	}
 }
 
+func IsUnsupportedBrowser(r *http.Request) bool {
+	userAgentHeader := r.Header.Get("User-Agent")
+	isUnsupported := false
+	unsupportedBrowserHeadersIdentifier := []string{
+		"Trident/", // IE 11
+		"MSIE ",    // IE 10 or older
+	}
+	for _, identifier := range unsupportedBrowserHeadersIdentifier {
+		if strings.Contains(userAgentHeader, identifier) {
+			isUnsupported = true
+			break
+		}
+	}
+	return isUnsupported
+}
+
+func SendUnsupportedBrowserResponse(rw http.ResponseWriter, brand string) {
+	content := `<p>Internet Explorer 11 and earlier are not supported, and the ` + brand + ` console will not function with your current browser.
+Please use a supported browser such as the latest version of Google Chrome, Mozilla Firefox, Microsoft Edge, or Safari.
+The current Firefox ESR (Extended Support Release) is also supported. To see the browsers we test, refer to the
+<a href="https://access.redhat.com/articles/4763741">OpenShift Container Platform 4.x Tested Integrations (for x86_x64)</a>.</p>`
```

Comment:
This looks good to me, although I know @rhamilto had concerns about being so specific in case what we support changes. I'm OK either way. @rhamilto ?

Reply:
I'm good with the consensus, whatever that may be.

Reply:
right, but your formulation is reasonable and make sense to me, so went with that.
@rhamilto thoughts?

Reply:
My take is less is more.  We know users don't read every word, and brevity is important when trying to relay something important.  So I'm a fan of significantly less text here, but am happy to roll with the majority opinion.